scripts/functions/requirements/pclinuxos

```diff
@@ -45,20 +57,21 @@ or there was problem checking if libyaml-devel is available / installed."
 # not sure if this one is needed for pclinuxos, copied from centos
 requirements_pclinuxos_update_system_check_epel()
```

Comment:
epel2016 does not exist. The latest is epel7, which I just added, but it's not gonna solve our issue.